plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -28,7 +28,25 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-NEVRA = namedtuple('NEVRA', ['name', 'epoch', 'version', 'release', 'arch'])
+_NEVRA = namedtuple('NEVRA', ['name', 'epoch', 'version', 'release', 'arch'])
+
+
+class NEVRA(_NEVRA):
+    @classmethod
+    def _fromdict(cls, nevra_dict):
+        """Given a dict with NEVRA keys, return a NEVRA namedtuple using its values
+
+        :param nevra_dict: dict with NEVRA keys (name, epoch, version, release, arch)
```

Comment:
I thought it worked that way as well, but in testing it looks like `namedtuple` will explode if you do this. I do want _fromdict to be able to take a dict that contains more than the nevra keys, so I think I'll update _fromdict to explicitly support this, and update the docstring like you mention here.